src/current/_includes/v23.1/essential-metrics.md

```diff
@@ -66,38 +66,38 @@ The **Usage** column explains why each metric is important to visualize in a cus
 
 ## KV distributed
 
-| <div style="width:225px">CockroachDB Metric Name</div> | {% if include.deployment == 'self-hosted' %}<div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb/?tab=host#metrics)<br>(add `cockroachdb.` prefix)</div> |{% elsif include.deployment == 'dedicated' %}<div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb_dedicated/#metrics)<br>(add `crdb_dedicated.` prefix)</div> |{% endif %}<div style="width:150px">Description</div>| Usage |
-| ----------------------------------------------------- | {% if include.deployment == 'self-hosted' %}------ |{% elsif include.deployment == 'dedicated' %}---- |{% endif %} ------------------------------------------------------------ | ------------------------------------------------------------ |
-| liveness.heartbeatlatency                           | {% if include.deployment == 'self-hosted' %}liveness.heartbeatlatency-p90 |{% elsif include.deployment == 'dedicated' %}liveness.heartbeatlatency |{% endif %} Node liveness heartbeat latency                              | If this metric exceeds 1 second, it is a sign of cluster instability. |
+| <div style="width:225px">CockroachDB Metric Name</div> | {% if include.deployment == 'self-hosted' %}<div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb/?tab=host#metrics)<br>(add `cockroachdb.` prefix)</div> |{% elsif include.deployment == 'advanced' %}<div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb_dedicated/#metrics)<br>(add `crdb_cloud.` prefix)</div> |{% endif %}<div style="width:150px">Description</div>| Usage |
```

Comment:
same question re: `crdb_dedicated` change to `crdb_cloud`

Reply:
PS I see there are a lot more mentions of this prefix change so I'll stop adding comments about it now :-)